Marlin/ubl_G29.cpp

```diff
@@ -311,8 +311,7 @@
   void unified_bed_leveling::G29() {
 
     if (!settings.calc_num_meshes()) {
-      SERIAL_PROTOCOLLNPGM("?You need to enable your EEPROM and initialize it");
-      SERIAL_PROTOCOLLNPGM("with M502, M500, M501 in that order.\n");
+      SERIAL_PROTOCOLLNPGM("?Enable EEPROM and init with M502, M500.\n");
```

Comment:
This error message confused me when I got it a moment ago. I was doing M502 and M500 but was still getting the message. When I added M501, it started working. How come it was removed from the helpful error message?

Reply:
There's no reason for `M501` to be needed. If `M501` is required something is broken. Someday someone will have to look into that.

Reply:
I see the rationale. UBL doesn't know where it can start storing its meshes until after `M501`. This is something that we can certainly figure out at compile time, but for now you do need to use `M501` as a workaround for this oversight.